README.md

```diff
@@ -24,31 +24,36 @@ GoRails should help you to get everything started.
 
 ### Getting Started
 
- 1. Download bundler to install dependencies
+ 1. We use submodules to manage themes. Update the default theme:
```

Comment:
It's not the theme, this is actually for the [pygments-css](https://github.com/richleland/pygments-css) repo. No need to mention themes, I think.